game/models.py

```
@@ -513,12 +514,31 @@ def is_expired(self):
     def __str__(self):
         return f"Game {self.id} ({self.white_label} vs {self.black_label})"
 
+def validate_game_state(value):
+    if not value:
+        return
+    if not isinstance(value, dict):
+        raise ValidationError("game_state must be a dictionary")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)